docs/autogenerated-menu/Our Practices/Engineering/Procedures/Document Demonstrate Duplicate.md

```diff
@@ -0,0 +1,41 @@
+# Document Demonstrate Duplicate
+
+Whether you’re bringing in a new starter, handing over to support, sharing knowledge across teams, or passing things back to the client.  Consistent documentation and training is the cornerstone to make sure the important stuff actually lands.
```

Comment:
```suggestion
Consistent documentation and training are the cornerstones of ensuring important stuff actually lands. Whether you’re bringing in a new starter, handing over to support, sharing knowledge across teams, or passing things back to the client, documentation and training are key.  
```

It doesn't read right as-is because starts with "whether" but the following clause is in a different sentence. Also uses "is" for multiple things so should be "are' and a cornerstone should be part of something - "of" for example.